tests/testutils/policy_controller_utils.go

```diff
@@ -32,5 +33,5 @@ func RunPolicyController(etcdIP, kconfigfile string) *containers.Container {
 		"-e", fmt.Sprintf("KUBECONFIG=%s", kconfigfile),
 		"-e", "RECONCILER_PERIOD=10s",
 		"-v", fmt.Sprintf("%s:%s", kconfigfile, kconfigfile),
-		"calico/kube-policy-controller:latest")
+		fmt.Sprintf("calico/kube-policy-controller%s:latest",os.Getenv("ARCHTAG")))
```

Comment:
I'd prefer if we interpolated this in the Makefile instead, which would allow users to run FV's against _any_ policy controller image, instead of just being able to flip which arch they want. How about using the existing `os.Getenv("CONTAINER_NAME")`?

Reply:
ok,  I will make those changes.  BTW:  felix already has the same issue,  I will attempt to get felix fixes in soon as well.